bootstrap/state-builder/src/build_balances.rs

```diff
@@ -59,13 +59,15 @@ fn main() {
 
         // Load the address. The UTXO dump tool we use doesn't output all the addresses
         // we support, so if parsing the address itself fails, we try parsing the script directly.
+        let bitcoin_newtork = into_bitcoin_network(args.network);
```

Comment:
Same typo here.   :slightly_smiling_face:

Reply:
done for all the occurrences.